src/StorSimple/Properties/AssemblyInfo.cs

```diff
@@ -21,7 +21,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure StorSimple management functions")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.2.0")]
```

Comment:
Done. It would be great if the owner of this file could update it though. It auto-updates so I have to explicitly revert it on every build.